src/components/Buttons/CloseChatWindowButton/CloseChatWindowButton.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import CloseChatWindowIcon from '../../../icons/CloseChatWindowIcon';
+import useChatContext from '../../../hooks/useChatContext/useChatContext';
+
+export default function CloseChatWindowButton() {
+  const { setIsChatWindowOpen } = useChatContext();
+
+  const closeChatWindow = () => {
+    setIsChatWindowOpen(false);
+  };
+
+  return (
+    <div onClick={closeChatWindow}>
```

Comment:
Looks like the close icon isn't perfectly centered. There seems to be a little extra vertical space in that div:

![image](https://user-images.githubusercontent.com/12685223/108401003-1ccd7180-71d9-11eb-832e-a8a003c20eda.png)

Seems that adding `display: flex` to the div fixes the issue. For a single css property, I think it's fine to just add it as a `style` prop:

`<div style={{ display: 'flex' }}>`

Reply:
nice find. because i moved all of this code to the `ChatWindowHeader` component, I was able to add this css to the existing `useStyles` object and avoid using the style prop